cli_tools/common/utils/test/test_utils.go

```diff
@@ -0,0 +1,87 @@
+//  Copyright 2019 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+//+build !test
```

Comment:
unsure if you use tags elsewhere, but what I have done here is just name the file test_utils_test.go so it only gets compiled in for tests

Reply:
Is there any downside to using build tags? 

I think I tried having _test file name ending, but then I was getting errors that the file doesn't contain any tests (which is true).

Reply:
If there are no downsides to build tags, I'd prefer to keep it as is: naming a file as a test to trick build is an uglier solution IMO.